persistence/src/main/scala/com/mesosphere/usi/persistence/KVRepositoryImpl.scala

```diff
@@ -0,0 +1,63 @@
+package com.mesosphere.usi.persistence
+
+import akka.{Done, NotUsed}
+import akka.stream.scaladsl.Source
+import com.mesosphere.usi.core.models.{PodRecord, ReservationRecord}
+import com.mesosphere.usi.persistence.storage.KVStorage
+import com.mesosphere.usi.persistence.serialization.SerializationFormats.{javaSerializationPodRecord, javaSerializationReservationRecord}
+import com.mesosphere.usi.persistence.serialization.Serialization
+
+import scala.concurrent.Future
+
+/**
+  * Implementation of the repository that uses an underlying key-value storage for storing records.
```

Comment:
Could you link respository
```suggestion
  * Implementation of the [[com.mesosphere.usi.persistence.Repository]] that uses an underlying key-value storage for storing records.
```